powerspikegg/go_gateway/computation.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"fmt"
+	"github.com/golang/protobuf/jsonpb"
+	"golang.org/x/net/context"
+	"net/http"
+	lolpb "powerspike.gg/powerspikegg/rawdata/public/leagueoflegends_gopb"
+	computation "powerspike.gg/powerspikegg/serving/match_computation_gopb"
+	"strconv"
+	"strings"
+)
+
+func getFeature(ctx context.Context, client computation.MatchComputationClient, summonerID int32, matchID int64, region lolpb.Region) (*computation.MatchComputationFeature, error) {
```

Comment:
I am kinda unsure why this is not a method of the gateway server struct since you use elements of the structure, and it is kinda related to the structure.